modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2991,6 +2993,25 @@ check-cron-expression=Check Cron Expression
+check-if-a-field-contains-a-specific-value-and-return-a-boolean-if-the-field-does-contain-the-value-it-is-invalid=Check if a field contains a specific value and return a Boolean. If the field does contain the value, it returns false. This function can be used for both text and numeric fields.
+check-if-a-field-has-the-same-date-of-the-value=Check if a field has the same date of the value. If the field has not the same value, it returns false.
+check-if-a-field-is-equal-to-a-specific-value-and-return-a-boolean=Check if a field is equal to a specific value and return a Boolean. If they are not equal, it returns false. This function can be used for both text and numeric fields.
+check-if-a-field-is-not-equal-to-a-specific-value-and-return-a-boolean=Check if a field is not equal to a specific value and return a Boolean. If it is not, the field returns false. This function can be used for both text and numeric fields.
+check-if-a-numeric-field-is-a-decimal-and-return-a-boolean=Check if a numeric field is a decimal and return a Boolean. If it is not, the function returns false.
+check-if-a-numeric-field-is-an-integer-and-return-a-boolean=Check if a numeric field is an integer and return a Boolean. If is not, the function returns false.
+check-if-a-numeric-field-is-greater-than-a-specific-numeric-value-and-return-a-boolean=Check if a numeric field is greater than a specific numeric value and return a Boolean. If it is not, the function returns false.
+check-if-a-numeric-field-is-greater-than-or-equal-to-a-specific-numeric-value-and-return-a-boolean=Check if a numeric field is greater than or equal to a specific numeric value and return a Boolean. If it is not, the function returns false.
+check-if-a-numeric-field-is-less-than-a-specific-numeric-value-and-return-a-boolean=Check if a numeric field is less than a specific numeric value and return a Boolean. If it is not, the function returns false.
+check-if-a-numeric-field-is-less-than-or-equal-to-a-specific-numeric-value-and-return-a-boolean=Check if a numeric field is less than or equal to a specific numeric value and return a Boolean. If it is not, the function returns false.
+check-if-a-text-field-is-a-URL-and-return-a-boolean=Check if a text field is a URL and return a Boolean. If the field does not match a URL pattern, it returns false. Valid entries must use standard URL elements (e.g., http, ://, .com)
+check-if-a-text-field-is-an-email-and-return-a-boolean=Check if a text field is an email and return a Boolean. If the field does not match a specific email or email pattern, it returns false. Valid entries must use standard email elements (e.g., @gmail, .com)
+check-if-a-text-field-is-different-to-a-specific-string-value-and-return-a-boolean=Check if a text field is different to a specific string value and return a Boolean. If they are equal, it returns false.
```

Comment:
@MariMalta I think this language key is not being used, could you check it, please?